tests/unit/test_model_card.py

```diff
@@ -1063,6 +1115,9 @@ def test_training_details_autodiscovery_from_job_name(session):
     assert len(training_details.training_job_details.training_metrics) == len(
         SEARCH_TRAINING_JOB_EXAMPLE["Results"][0]["TrainingJob"]["FinalMetricDataList"]
     )
+    assert len(training_details.training_job_details.training_metrics) == len(
```

Comment:
why `training_details.training_job_details.training_metrics` ?

Reply:
Will change training_metrics to hyper_parameters.